ITEM.tsv

```diff
@@ -22,11 +22,11 @@ ITEM_20150317_000021	$Recommended for those who desire a heavier and stronger sw
 ITEM_20150317_000022	$Sabre 
 ITEM_20150317_000023	$Primary weapon of the Cavalry, optimized for slashing. The guard on the hilt helps protect the user's hand. 
 ITEM_20150317_000024	$Shamshir 
-ITEM_20150317_000025	$The large curved crescent edge allows for a powerful slash. 
+ITEM_20150317_000025	$A blade that has a long, curved crescent. It dishes out heavy slashes.
 ITEM_20150317_000026	$Scimitar 
-ITEM_20150317_000027	$Its heavy weight is compensated by its very powerful slash. 
+ITEM_20150317_000027	$Its heavy weight is compensated by its very powerful slashes. 
 ITEM_20150317_000028	$Kris 
-ITEM_20150317_000029	$The twisted edge amplifies the damage it inflicts. A sword that is both practical and decorative. 
+ITEM_20150317_000029	$A snakelike sword that is both practical and decorative. 
```

Comment:
Ah, I guess it's a personal preference thing for me when it comes to things that "Amplify" or have "Powerful" in the description text. If it actually comes with a bonus to those things... Such as something that increases damage to Beast mobs, it could say "Amplifies the damage it inflicts to beast-type monsters." but if it's just a basic weapon without the tertiary attributes, it shouldn't say that it's "Powerful" or that it "Amplifies" anything, etc.? 

Kind of like picking up a Lv. 1 sword that has "Powerful Slashes" and then a Lv. 150 sword that "Has a dull edge" etc. Not sure if you're following me :astonished: